components/scream/src/share/io/scream_scorpio_interface.F90

```diff
@@ -637,17 +637,21 @@ subroutine eam_pio_createHeader(File)
     integer                       :: retval
 
     ! TODO change options below to match specific simulation case
-    retval=pio_put_att (File, PIO_GLOBAL, 'source', 'SCREAM')
-    retval=pio_put_att (File, PIO_GLOBAL, 'case', 'TEST 1')
-    retval=pio_put_att (File, PIO_GLOBAL, 'title', 'SCORPIO TEST')
-    retval=pio_put_att (File, PIO_GLOBAL, 'logname','THE GIT LOG HASH')
-    retval=pio_put_att (File, PIO_GLOBAL, 'host', 'THE HOST')
-    retval=pio_put_att (File, PIO_GLOBAL, 'Version', &
-           '0')
-    retval=pio_put_att (File, PIO_GLOBAL, 'revision_Id', &
-           'None')
-    retval=pio_put_att (File, PIO_GLOBAL, 'initial_file', 'NONE FOR NOW')
-    retval=pio_put_att (File, PIO_GLOBAL, 'topography_file', 'NONE FOR NOW')
+    retval=pio_put_att (File, PIO_GLOBAL, 'source', 'E3SM Atmosphere Model Version 4')
```

Comment:
@PeterCaldwell , thank you for doing this.  It has bothered me for awhile but I've kept putting off the change.